spec/setup.lua

```diff
@@ -15,4 +21,7 @@ _G.Handlers = {
 	},
 }
 
+-- setup all process globals
+require("src/globals")
```

Comment:
Would it not make sense to go through init.lua?

Reply:
or perhaps process?

Reply:
update here - https://github.com/ar-io/ar-io-network-process/pull/397

Reply:
Wrong link?

Reply:
sorry - here https://github.com/ar-io/ar-io-network-process/pull/384/commits/cc823482b6466d7a66f77e92fbd39609665cc006